deploy/configs/externals/modules.yaml

```diff
@@ -29,6 +29,7 @@ modules:
     whitelist:
       - arm-forge
       - blender
+      - byobu
```

Comment:
Is this used by `end users`?
Just a note : as a developer, one can now do `spack load byobu` to load the package. It's not necessary to have module.

Reply:
yes, requested by several viz team members (not spack developers)

Reply:
I don't think that this is a good idea. These are for the end users enjoyment, not an absolute necessity. As such, I'd normally say install the binary build like for other software (neovim, starship, ytop…) into `${HOME}/.local`.

I would maybe agree to this if there is a module akin to `python-dev` that spans several of these utilities.

Reply:
I'd not say this is merely for end-users enjoyment when everybody works on VPN and experience some connections issues.

Let's discuss it tomorrow during Deployment sync meeting, but I don't really see it feasible for non-technical people to install it themselves. Myself I have several packages/modules in my ${HOME} but the effort to train non-technical/sporadic bb5 users (on building with spack) doesn't seem to be worth it.